### PR TITLE
docs: add Wallaby.js to IDE Integration page

### PR DESCRIPTION
**Pull request by smcenlly**

### CODE REVIEW

**Comment by sheremet-va** [comment]
Docs build is failing with:

[vite]: Rollup failed to resolve import "/wallabyjs.com/assets/img/vitest_cover.png" from "guide/ide.md".

**Comment by smcenlly** [comment]
Looks like netlify checks failed. I've not used netlify before and am not entirely sure how to fix. Is someone from the vitest team able to tell me how to fix (or else fix it for me)?

File: docs/guide/ide.md [hunk 1]
```diff
@@ -25,3 +25,18 @@ Maintained by [@linux-china](https://github.com/linux-china).
...
 ![Vitest WebStorm Screenshot](https://raw.githubusercontent.com/linux-china/vitest-jetbrains-plugin/main/screenshot.png)
+
+## Wallaby.js <sup><code>Paid (free for OSS)</code></sup>
+
+Created by [The Wallaby Team](https://wallabyjs.com)
+
+[Wallaby.js](https://wallabyjs.com) runs your Vitest tests immediately as you type, highlighting results in your IDE right next to your code.
+
+<p text-left>
+<img src="https:/wallabyjs.com/assets/img/vitest_cover.png" w-142 />
```
Comment (Demivan) [inline docs/guide/ide.md:36]:
```suggestion
<img src="https://wallabyjs.com/assets/img/vitest_cover.png" w-142 />
```

File: docs/guide/ide.md [hunk 2]
```diff
@@ -25,3 +25,18 @@ Maintained by [@linux-china](https://github.com/linux-china).
...
+</p>
+
+[VS Code](https://marketplace.visualstudio.com/items?itemName=WallabyJs.wallaby-vscode) | [JetBrains](https://plugins.jetbrains.com/plugin/15742-wallaby) |
+[Visual Studio](https://marketplace.visualstudio.com/items?itemName=vs-publisher-999439.WallabyjsforVisualStudio2022) | [Sublime Text](https://packagecontrol.io/packages/Wallaby)
+
+![Wallaby VS Code Demo](https:/wallabyjs.com/assets/img/vitest_demo.gif)
```
Comment (Demivan) [inline docs/guide/ide.md:42]:
```suggestion
![Wallaby VS Code Demo](https://wallabyjs.com/assets/img/vitest_demo.gif)
```

**Review by Demivan — CHANGES_REQUESTED** [review]

**Comment by smcenlly** [comment]
Thanks!

**Comment by Demivan** [comment]
https://deploy-preview-1977--vitest-dev.netlify.app/guide/ide.html - preview url

**Review by Demivan — APPROVED** [review]

**Pull request merged**